command/snapshot/inspect/formatter.go

```diff
@@ -57,9 +57,22 @@ func (_ *prettyFormatter) Format(info *OutputFormat) (string, error) {
 	fmt.Fprintf(tw, "\n %s\t%s\t%s\t", "----", "----", "----")
 	fmt.Fprintf(tw, "\n Total\t\t%s\t", ByteSize(uint64(info.TotalSize)))
 
+	if detailed {
```

Comment:
If we no longer have detailed then we can just do a not nil check here on `info.StatsKV`, might need some fangaling to work